build-caching/pom.xml

```diff
@@ -23,6 +23,13 @@
             <version>0.15.0</version>
         </dependency>
 
+        <!-- JSON serialization -->
+        <dependency>
```

Comment:
probably should junk this in favor of protobufs, but it does have the advantage of already being included via other dependencies, so we're not adding to our classpath.